### PR TITLE
Fix dog armor being too small for the largest dogs

### LINKED ISSUE #46288: Can't equip armor on Great Pyrenees
Can't equip Grand pyrenees with armor. (No adecuate item at hand) Other dog breeds seem to work fine except the rotweiler. Considering the armor is tailor-made, it shouldn't matter the size of the dog. If it does matter, the message should be more clear. Spawned another dog of the same breed and all different dog armors just in case my dog in particular was bugged (it wasn't healing while it was in my reality bubble), but it seems to be only related to the breed.

No dog seems capable of equiping the biosilified harness, it says it is too heavy for them.

### PR DESCRIPTION
**Pull request by hexagonrecursion**
#### Summary

SUMMARY: Content "Allow Great Pyrenees and rottweiler to wear dog armor"

#### Purpose of change

Fixes #46288

#### Describe the solution

As suggested here https://github.com/CleverRaven/Cataclysm-DDA/issues/46288#issuecomment-751498200 I increased the max_pet_vol to include the largest dogs.

#### Describe alternatives you've considered

See https://github.com/CleverRaven/Cataclysm-DDA/issues/46288#issuecomment-751498200

#### Testing

1. The game loads
2. I can equip Great Pyrenees with Kevlar dog harness

#### Additional context

none

### CODE REVIEW

File: data/json/items/tool/workshop.json [hunk 1]
```diff
@@ -93,15 +93,15 @@
     "name": { "str": "brick kiln" },
     "description": "This is a portable charcoal-fired kiln.  It is designed for firing bricks, but you could use it to fire anything made of clay.",
     "weight": "9600 g",
-    "volume": "9 L",
```
Comment (Rivet-the-Zombie) [inline data/json/items/tool/workshop.json:96]: Unrelated commit?

Comment (hexagonrecursion) [reply data/json/items/tool/workshop.json]: Sorry. Good catch.

**Review by BrettDong — APPROVED** [review]
